plugins/statistics.koplugin/calendarview.lua

```diff
@@ -523,25 +528,27 @@ function CalendarView:init()
                 })
             end,
         },
+        call_hold_input_on_tap = true,
         bordersize = 0,
-        margin = Screen:scaleBySize(20),
```

Comment:
Removing this may cause some stuff to go to near the footer (like previously with KeyValuePage - where I removed some stuff from avalable_height to compensate for this loss).